sql/size_utils.sql

```diff
@@ -374,7 +373,7 @@ BEGIN
         WHERE c.OID = main_table;
 
         RETURN QUERY
-        SELECT h.schema_name || '.' || ci.hypertable_index_name,
+        SELECT format('%I.%I', h.schema_name, ci.hypertable_index_name),
```

Comment:
Why single `%` here but not above?

Reply:
It is not nested inside another format so it is not needed.